### ISSUE TITLE
Link to the downloads page is broken

### ISSUE THREAD

**Issue by germanferrero** [RetroShare/documentation#77]
At https://retroshare.readthedocs.io/en/latest/user-guide/installation/
the link to the downloads page is: https://retroshare.net/downloads.html
should we change it to https://retroshare.cc/downloads.html ?
It doesn't redirect automatically.

**Comment by G10h4ck** [comment]
yeah either we remove that page in the documentation, or we update to the correct address that is the retroshare.cc one

**Comment by datenbrei** [comment]
It looks like the documentation source is up-to-date but the deployed documentation is out of date or not updated yet.

**Comment by G10h4ck** [comment]
Don't know how to force readthedocs to update the deployed documentation @cavebeat is the one who setup that

**Comment by datenbrei** [comment]
@cavebeat Maybe, it can be automated, triggered after changes in the source? The source should be the controlled part, so this would be ok, I think.

**Comment by Drea5er** [comment]
It looks like the deployed documents on ReadtheDocs  is still not synced with the current source code.
This Download link for example still seem to show the old link, NOT reflecting what is in the Source Code. 
Did @cavebeat  know how to Automate as per @datenbrei suggestion?
